redash/templates/emails/reset.html

```diff
@@ -2,16 +2,34 @@
 
 {% block content %}
 
-<p class="intercom-align-left" style="line-height: 1.5; margin: 0 0 17px; text-align: left !important" align="left">{{_('Hi')}} {{ user.name }},</p>
-<h2 class="intercom-align-left" style="color: #282F33; font-size: 18px; font-weight: bold; margin-bottom: 7px; margin-top: 30px; text-align: left !important" align="left">
-    {{ _("You told us you forgot your password. If you really did, click here to choose a new one:") }}
-</h2>
-<table class="intercom-container intercom-align-center" align="center" style="border-collapse: collapse; border-spacing: 0; margin: 17px auto; table-layout: fixed; text-align: center !important"><tr><td style="background: #0071b2; border: 1px none #dadada; border-radius: 3px; font-family: Helvetica, Arial, sans-serif; font-size: 16px; margin: 0; padding: 12px 35px; text-align: left; vertical-align: top" align="left" bgcolor="#0071b2" valign="top"><a class="intercom-h2b-button" target="_blank" href="{{ reset_link }}" style="background: #0071b2; border: none; border-radius: 3px; color: white; display: inline-block; font-size: 14px; font-weight: bold; outline: none !important; text-decoration: none">{{ ("Choose a new password") }}</a></td></tr></table>
+<p class="intercom-align-left" style="line-height: 1.5; margin: 0 0 17px; text-align: left !important" align="left">Hallo {{ user.name }},</p>
```

Comment:
I am guessing that these templates don't come with their own css files otherwise you'd have already changed it there by now :D